compiler/rustc_codegen_llvm/src/gotoc/hooks.rs

```diff
@@ -322,6 +322,7 @@ impl<'tcx> GotocHook<'tcx> for MemSwap {
             block.push(ye.dereference().assign(var, loc.clone()));
 
             Symbol::function(
+                &func_name,
```

Comment:
Do we want the same thing for the pretty-name and name?

Reply:
Probably not, after offline discussion, removing and passing through as an option/None